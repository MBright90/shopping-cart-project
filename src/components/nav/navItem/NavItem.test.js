import { render } from '@testing-library/react'
import React from 'react'
import { HashRouter } from 'react-router-dom'

import NavItem from './index'

describe('NavItem', () => {
  const linkDestination = 'testDestination'
  let props

  beforeEach(() => {
    props = {
      linkDestination
    }
  })

  it('renders to match snapshot', () => {
    const { container } = render(<NavItem {...props} />, {
      wrapper: ({ children }) => <HashRouter basename="/">{children}</HashRouter>
    })
    expect(container).toMatchSnapshot()
  })
})
