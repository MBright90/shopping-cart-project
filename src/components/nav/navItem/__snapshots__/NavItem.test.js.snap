// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem renders to match snapshot 1`] = `
<div>
  <li
    class="navItem"
  >
    <a
      href="#/testdestination"
    >
      testDestination
    </a>
  </li>
</div>
`;
